mitodl/edx-sga
# Fixed size of iframe on modal opening

#### What are the relevant tickets?
Fixes the LMS issue for ticket #308 

#### What's this PR do?
This PR updated the iframe size on modal opening and closing

#### How should this be manually tested?
This  PR can be manually tested by following steps
    - Create a SGA unit in Studio
    - Open the unit on LMS with a staff user
    - Press Grade submissions button and verify popup is not cutoff

#### Screenshots (if appropriate)
(Optional)

<img width="1434" alt="Screenshot 2021-12-22 at 5 50 03 PM" src="https://user-images.githubusercontent.com/87968618/147097534-7f088313-1e3a-4399-b8a3-feebfa641ead.png">

<img width="1434" alt="Screenshot 2021-12-22 at 5 49 57 PM" src="https://user-images.githubusercontent.com/87968618/147097948-2ef2cb84-c11c-4872-90d7-cc95069188d2.png">
